src/redux/sagas/motions/voteMotion.ts

```diff
@@ -71,19 +67,7 @@ function* voteMotion({
       'voteMotionTransaction',
```

Comment:
Why do we add this message to metamask? (expand code above: "Sign this message to generate 'salt' entropy.")

It's currently too long for the gas stn so either I adjust the css there or we remove.

Reply:
@rdig

Reply:
Really, we remove it? :)

And how would the user know what he's signing then ?

The message is needed as it acts like the string the user is signing to prove their identity when voting and revealing. 

The problem here is the gas station, as it should be able to support any message length. However, since in it's current form it's not really long for this world anymore, we can just fix it for this instance, via CSS

Reply:
Thanks. Will fix in the gas stn.